### DISCUSSION TITLE
spyne.model.complex.XmlAttribute issue

### DISCUSSION THREAD

**Discussion by minafarouk** [arskom/spyne#671] [Q&A] [answered]
I am trying to upgrade spyne from **2.10.9** to **2.13.16** but I got error: "`TypeError: __new__() got an unexpected keyword argument 'attribute_of`' " for

```
class AVPairList(ComplexModel):
    AVPair = String(max_occurs='unbounded')
    Name = XmlAttribute(String, attribute_of='AVPair')
```

**Comment by gopackgo90** [comment]
`attribute_of` was removed in 2.13.3-alpha, see https://github.com/arskom/spyne/commit/2a483f34b1a50ccb35e7370ff86cb599281b7cbb.

**Comment by plq** [comment, marked as answer]
I just modernized the xml utils example. See the ``DateTimeWithAttribute`` class in https://github.com/plq/spyne/blob/d27b42a50166be6b480772134e96307e3438ad9f/examples/xml/utils.py
